rest/server_context.go

```diff
@@ -284,6 +285,10 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 		viewQueryTimeoutSecs = config.ViewQueryTimeoutSecs
 	}
 
+	if config.OldRevExpirySeconds != nil && *config.OldRevExpirySeconds > 0 {
+		oldRevExpirySeconds = int(*config.OldRevExpirySeconds)
+	}
```

Comment:
I feel like setting an explicit expiry value of zero in the config should represent something (since we're already treating the absence of a setting as 'use default'.  The normal use of expiry=0 would be 'no expiry' so I guess that's what we should be doing here, although that would be an uncommon use case.

The more likely use case would be 'no backup', but I think having expiry=0 mean 'no backup' would be counter-intuitive.

Reply:
A config value of zero is now passed through to the bucket calls. From the CBS docs:
```
Time to live (TTL) is the amount of time until a document expires in Couchbase Server. By default, all documents have a TTL of zero, which indicates the document is kept indefinitely. 
```